pkg/util/util.go

```diff
@@ -782,12 +782,11 @@ func SafeGetBool(b *bool) bool {
 
 // IsPortFree checks if the port on localhost is free to use
 func IsPortFree(port int) bool {
-	address := fmt.Sprintf("localhost:%d", port)
+	address := fmt.Sprintf("0.0.0.0:%d", port)
```

Comment:
Should this change be reverted once the regression has been fixed?  If so, can you add a todo about it?

Reply:
To me, this is still valid even after the regression is fixed in the kernel. Podman listens by default on `0.0.0.0`, so it makes sense to check on all addresses. As for Kubernetes, port-forwarding currently binds to  `127.0.0.1` by default, but it makes sense to me to still check on all addresses.
When https://github.com/redhat-developer/odo/issues/6479 is implemented, we would need to properly support the address on which to bind and pass the right address to this function.